components/Button.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+export const Button = ({
+  text,
+  defaultClass,
+  hoverClass,
+  href,
+  isExternal,
+}: {
+  text: string;
+  defaultClass: string;
```

Comment:
instead of providing defaultClass and hoverClass each time we use the component, it's a common approach to support variants for buttons. I see currently there are three (or four) variants (can be discussed with Roman) - the black one, the transparent one (with black or green text/border) and the white one (in products section)
would be great to just easily use it as something like

```
<Button type="secondary" disabled/>
```

or 
```
<Button type="primary" asChild>
   <Link href.... />
</Button>
```